pkg/web/oauthclient/config.go

```diff
@@ -24,6 +24,8 @@ type Config struct {
 	ClientID     string `name:"client-id" description:"The OAuth client ID"`
 	ClientSecret string `name:"client-secret" description:"The OAuth client secret" json:"-"`
 
+	CrossSiteCookie bool `name:"cross-site-cookie" description:"Whether to make OAuth cookies accessible cross-site"`
```

Comment:
This needs to be mentioned in the CHANGELOG, also the fact that multi-cluster deployments need to set this to `true` in order to support OAuth clients that use POST callbacks. Please also create an issue for setting that to `true` in https://github.com/TheThingsIndustries/lorawan-stack-aws (doesn't need to be configurable, since those deployments always use HTTPS).